tests/route/test_route_consistency.py

```diff
@@ -16,6 +16,27 @@
 
 logger = logging.getLogger(__name__)
 
+@pytest.fixture(scope='function', autouse=True)
+def check_and_disable_routecheck(duthosts):
+    # Previous test can create route inconsistency for 1-2 cycle
+    # and anoter cycle as part of this test can cause failure of this test because of monit ERR.        
+    # To mitigate this make sure route_check is clean before starting the test and if so than restart
+    # monit route check to reset it's error check counter
+    for duthost in duthosts:
```

Comment:
Since the function could be used in wide scope and before every function in the future.
monit restart could be fast, but route_check could be slow
Can we use SafeThreadPoolExecutor to run them in parallel?